RELEASE/scripts/autoscend/combat/auto_combat_default_stage4.ash

```diff
@@ -124,6 +124,20 @@ string auto_combatDefaultStage4(int round, monster enemy, string text)
 		return useItem($item[Abstraction: Action]);
 	}
 	
+	//these loofah skills stagger and provide MP, meat, or XP
```

Comment:
Would be good to add canSurvive checks too. Don't want to die because we tried to get some extra bonus of something. Perhaps canSurvive(4) to cast if we can survive 4 rounds?

Reply:
If they always stagger, it seems to me there's nothing to have to survive.  Am I mistaken?